spec/features/admin_adds_a_group_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     fill_in Group.human_attribute_name(:name), with: "St. Peter"
     fill_in Group.human_attribute_name(:responsable_name), with: "Mr. Black"
-    fill_in Group.human_attribute_name(:responsable_contact), with: "0045-123-4445"
+    fill_in Group.human_attribute_name(:responsable_contact), with: "0045-12344"
```

Comment:
Not sure where my comment disappeared. I suggested using the field's `id` (`group_responsible_contact`), or `name` (`group["resonible_contact"]`) attributes instead of `Group.human_attribute_name(:responsable_contact)`.

Which will then make it easy to fill it a phone number that is inline with real phone numbers.
If you now add a validation for length of the phone number, this spec will fail.
We always want to fill in details as close to reality as we can in our specs.